components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/NetworkMessageEncoder.cs

```diff
@@ -89,9 +97,8 @@ public class NetworkMessageEncoder : IMessageEncoder {
             var current = notifications.GetEnumerator();
             var processing = current.MoveNext();
             var messageSize = 2; // array brackets
-            maxMessageSize -= 2048; // reserve 2k for header
```

Comment:
>maxMessageSize -= 2048; [](start = 12, length = 23)

why deleted this line? we need to reserve space for header in payload

Reply:
Same as above